sdk/communication/azure-communication-chat/pom.xml

```diff
@@ -50,12 +50,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```

Comment:
revert? as we are not releasing chat

Reply:
Wouldn't compile. It has to be consistent with current